backend/graph/api/users.utils.go

```diff
@@ -58,3 +66,28 @@ func (r *Resolver) addItemToCollection(ctx context.Context, itemType string, ite
 	r.Loaders.UserCollectionEntryIDsLoader.Clear(ctx, myListID)
 	return entry, nil
 }
+
+func (r *Resolver) isInMyList(ctx context.Context, id uuid.UUID) (bool, error) {
+	myList, err := r.QueryRoot().MyList(ctx)
+	if err != nil {
+		return false, nil
+	}
+	list, err := r.Loaders.UserCollectionEntryIDsLoader.Get(ctx, utils.AsUuid(myList.ID))
+	if err != nil {
+		return false, err
+	}
+	entryIDs := utils.PointerArrayToArray(list)
+	chunks := lo.Chunk(entryIDs, 20)
+	for _, chunk := range chunks {
+		entries, err := r.Loaders.UserCollectionEntryLoader.GetMany(ctx, chunk)
```

Comment:
Maybe have `UserCollectionEntryIDsLoader` return the ItemIds instead of their "id"?
if only to make the code simpler

Reply:
Maybe in a refactor later.

Not sure why I have an ID for the entries. Could've just been a unique key with collection_id, type and item_id.

Reply:
Possibly to enable having the same track multiple times in a playlist. That's a very niche requirement tho, but spotify has it